src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -42,19 +42,19 @@ components:
       description: The SQS message for sending an application to print.
       type: object
       properties:
-        voterApplicationId:
+        sourceReference:
           type: string
-          format: uuid
-          description: The Voter Authority Certificate identifier
-          example: 14ab70e8-bd3b-400f-bd95-246caf9e4810
+          description: Reference in the source application that this print request relates to
         applicationReference:
           type: string
           description: The application reference as known by the citizen. Not guaranteed to be unique.
+        sourceType:
+          $ref: '#/components/schemas/SourceType'
         gssCode:
           type: string
           minLength: 9
           maxLength: 9
-          description: GSS Code to ensure permissions can be maintained in Print Service API
+          description: GSS code of the ERO responsible for sending the message
         issuingAuthority:
```

Comment:
`issuingAuthority` - eg: "Camden Borough Council". This is printed on the certificate.
There is an open question that @kristerbone is looking to resolve with the business .... given that we need to support dual language do we need `issuingAuthorityEnglish` and `issuingAuthorityWelsh` ?
Alternatively (and better?), given that we already have `eroEnglish` and `eroWelsh` that have a `name` property, do we need `issuingAuthority` at all? Is the Issuing Authority the same as the ERO name ?

@kristerbone is looking to resolve this question , but I'm not sure what to do about in the immediate term. Half of me says "leave it and we'll update it when we have the answers to the above"; and the other half of me says "delete it now on the basis that I'm hoping the Issuing Authority is the same as the ERO name, as we can always add it back (along with the Welsh version) if we need to"

Thoughts @kristerbone ?

Reply:
I will email about this today.

Reply:
OK, I'll leave it in for now so we can merge this PR and move on. As and when we have an answer to this we can update this message type as necessary 👍